### PR TITLE
fix: add dilapitan

### LINKED ISSUE #1422: Dominic Lapitan
## Profile Screenshot

dilapitan

### PR DESCRIPTION
**Pull request by dilapitan**
resolves #1422

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/dilapitan.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/23071809/193412365-2a2d6fb6-6284-4390-bfb1-696f6d605067.png",
+  "issueId": 1422,
+  "name": "Dominic Lapitan",
+  "username": "dilapitan"
+}
```
Comment (krshkodes) [inline profiles/dilapitan.json:6]:
```suggestion
}

```

**Review by krshkodes — CHANGES_REQUESTED** [review]

File: profiles/dilapitan.json (hunk 1 above)
Comment (dilapitan) [reply profiles/dilapitan.json]: Whoops, sorry I forgot about the last new line. Thanks!

**Review by krshkodes — APPROVED** [review]
Thank you!

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**